app/lib/frontend/handlers/pubapi.client.dart

```diff
@@ -18,7 +18,7 @@ export 'package:api_builder/_client_utils.dart' show RequestException;
 
 /// Client for invoking `PubApi` through the generated router.
 ///
-/// Reponses other than 2xx causes the methods to throw
+/// Responses other than 2xx causes the methods to throw
```

Comment:
This file is generated code, not where this should be fixed.

Reply:
For my reference the commit also changed the generator: https://github.com/jsoref/dart-lang-pub-dev/commit/f45d4d1d216dddbd1c5e4e684541b34271231cda#diff-f3b5b16c84fd9774864c109d5728058f3b157665c8915759d8a7327eb7a0cbe4